backend/app/adapter/gqlapi/resolver/error.go

```diff
@@ -160,3 +161,22 @@ func (e ErrMaliciousContent) Extensions() map[string]interface{} {
 func (e ErrMaliciousContent) Error() string {
 	return "contains malicious content"
 }
+
+// ErrUnauthorizedAction signifies the requesting user is not authorized for the action.
```

Comment:
// ErrUnauthorizedAction signifies the requesting user is not allowed to perform certain action.

Reply:
Forgot to leave this comment.

Reply:
@byliuyang , I have addressed this in #846, since it is trivial to open a new PR.